engine/src/main/java/org/hibernate/validator/internal/util/StringHelper.java

```diff
@@ -81,7 +82,7 @@ public static String decapitalize(String string) {
 			return string;
 		}
 		else {
-			return string.substring( 0, 1 ).toLowerCase() + string.substring( 1 );
+			return string.substring( 0, 1 ).toLowerCase( Locale.ENGLISH ) + string.substring( 1 );
```

Comment:
After giving it some thoughts, could you change it to use Locale.ROOT? I think it emphasizes better what we want to do.

After that, we're good to go!

Thanks!